server/src/main/java/org/apache/kafka/server/metrics/ClientMetricsConfigs.java

```diff
@@ -133,8 +134,8 @@ private static void validateProperties(Properties properties) {
 
         // Make sure that push interval is between 100ms and 1 hour.
         if (properties.containsKey(PUSH_INTERVAL_MS)) {
-            int pushIntervalMs = Integer.parseInt(properties.getProperty(PUSH_INTERVAL_MS));
-            if (pushIntervalMs < MIN_INTERVAL_MS || pushIntervalMs > MAX_INTERVAL_MS) {
+            Integer pushIntervalMs = (Integer) ConfigDef.parseType(PUSH_INTERVAL_MS, properties.getProperty(PUSH_INTERVAL_MS), Type.INT);
+            if (pushIntervalMs == null || pushIntervalMs < MIN_INTERVAL_MS || pushIntervalMs > MAX_INTERVAL_MS) {
```

Comment:
Hmmm, what happens when we use `kafka-client-metrics.sh` tool to just update pattern or metrics i.e. pushIntervalMs is not updated? Then will pushIntervalMs is not null?

Reply:
Hi @apoorvmittal10, thanks for the review.

If kafka-client-metrics.sh doesn't have pushIntervalMs, it should not enter this block (check in 137). 
I added a null check just to ensure that null values are handled properly. 
Ideally, the check should always pass.